Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -150,7 +182,6 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
         removeCompletion()
         super.deleteBackward()
         enteredTextLength = count(self.text)
-        autocompleteDelegate?.autocompleteTextField(self, didEnterText: self.text)
```

Comment:
@thebnich I think we can safely remove this line. On master branch we are using `insertText` which calls only if we are entering text not deleting it. Here we are using `textChange` event and there we are notifying all change. However that will call with `debounce` delay. I do not think that is a problem.